sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/geojson/GeoArray.java

```diff
@@ -67,84 +68,7 @@ public int size() {
 
     @Override
     public Iterator<T> iterator() {
-        return new GeoArrayIterator();
-    }
-
-    @Override
-    public ListIterator<T> listIterator(int index) {
-        return new GeoArrayListIterator(index);
-    }
-
-    /**
-     * Throws {@link UnsupportedOperationException} as GeoArray doesn't support mutation.
-     *
-     * @param t The element that would be added.
-     * @return Throws an exception.
-     * @throws UnsupportedOperationException GeoArray doesn't support mutation.
-     */
-    @Override
-    public boolean add(T t) {
```

Comment:
Why are these methods removed? We still don't want any mutation on this type right?

Reply:
Yes, and the default implementation in `AbstractList` already throws.